modules/geoip-lite-2024-07-19/lib/utils.js

```
@@ -0,0 +1,98 @@
+	}
+
+	return r;
+};
+
+
+utils.cmp = function(a, b) {
+	if (typeof a === 'number' && typeof b === 'number') {
+		return (a < b ? -1 : (a > b ? 1 : 0));
+	}
+
+	if (a instanceof Array && b instanceof Array) {
```

Biome

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)